applications/nrf5340_audio/doc/feature_support.rst

```diff
@@ -31,3 +31,6 @@ The following QDIDs are related to the nRF5340 Audio application:
 .. include:: /libraries/bin/bt_ll_acs_nrf53/index.rst
    :start-after: le_audio_controller_qdid_start
    :end-before: le_audio_controller_qdid_end
+
+.. note::
+   As the LE Audio Controller for nRF5340 is deprecated in NCSDK 2.6.0 release, the software combination required for this QDID will no longer be supported.
```

Comment:
```suggestion

.. note::
   As the LE Audio Controller Subsystem for nRF5340 is deprecated in NCSDK 2.6.0 release, the software combination required for this QDID will no longer be supported. This controller will keep its respective QDID.
```

Reply:
```suggestion
.. note::
   Because the LE Audio controller for nRF5340 has been :ref:`deprecated <api_deprecation>` in the |NCS| v2.6.0, the software combination required for this QDID is no longer supported. This controller will keep its QDID, but no further qualifications will be made.
```
@koffes , can this work?

Also, @andvib , please move this note to in between lines 26-27 in the LE audio controller's index.rst, so that we can pull it in with the include and have these lines also in that file.

![image](https://github.com/nrfconnect/sdk-nrf/assets/41291385/8bae5ce5-867b-4810-83d4-f6364590518a)